src/components/share/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+import React, {Component} from 'react';
+
+class Share extends Component {
+    render() {
+        return (<div className='share-wrapper'>
+                    <span>Share</span>
+                    {this._tw()}
+                    {this._fb()}
+                </div>
+            );
+    }
+
+    _tw() {
```

Comment:
Url and function name are switched. _tw calls facebook sharer and _fb calls twitter intent XD